pkg/api/operators/apis/experiment/v1alpha2/experiment_types.go

```diff
@@ -35,10 +35,14 @@ type ExperimentSpec struct {
 	TrialTemplate *TrialTemplate `json:"trialTemplate,omitempty"`
 
 	// How many trials can be processed in parallel.
-	ParallelTrialCount int `json:"parallelTrialCount,omitempty"`
+	// Defaults to 3
+	ParallelTrialCount *int `json:"parallelTrialCount,omitempty"`
 
-	// Total number of trials to run.
-	MaxTrialCount int `json:"maxTrialCount,omitempty"`
+	// Max completed trials to mark experiment as succeeded
+	MaxTrialCount *int `json:"maxTrialCount,omitempty"`
```

Comment:
Name as MinTrialCount? Minimum trail count for success.. 
BTW, what is the benefit of pointer *int rather than int, which is changed in several places?